Core/Lib/Extensions/ApplicationSetupBuilderExtensions.cs

```diff
@@ -56,5 +58,52 @@ public static IApplicationSetupBuilder AddOAuthClient(this IApplicationSetupBuil
 
             return applicationSetup;
         }
+
+        /// <summary>
+        /// Add a HttpClient Service that gets a bearer-token from the configured IdentityServer
+        /// </summary>
+        /// <typeparam name="TClient"></typeparam>
```

Comment:
Remove the params and the returns tags if no description added. Summary is enough.

Reply:
Fixed